packages/nx/src/command-line/generate.ts

```diff
@@ -92,7 +97,9 @@ function throwInvalidInvocation() {
 }
 
 function readDefaultCollection(nxConfig: NxJsonConfiguration) {
```

Comment:
nit: might be cleater if we pluralize the function name, we now return multiple collections
```suggestion
function readDefaultCollections(nxConfig: NxJsonConfiguration) {
```